src/main/java/uk/gov/hmcts/reform/et/syaapi/service/InvalidAcasNumbersException.java

```diff
@@ -0,0 +1,33 @@
+package uk.gov.hmcts.reform.et.syaapi.service;
+
+import java.util.Arrays;
+
+/**
+ * Is thrown when errors are found in ACAS numbers in order to pinpoint the details.
+ */
+public class InvalidAcasNumbersException extends Exception {
+
+    private static final long serialVersionUID = -3042681110164047285L;
+    private final String[] invalidAcasNumbers;
```

Comment:
Any reason for using String array rather than collections?

Reply:
Agreed to keep as String array